18F/identity-idp
# Render icon for mobile navigation close button

## 🛠 Summary of changes

Updates rendering of the mobile navigation menu close button icon to use `IconComponent` instead of a custom SVG.

**Why?**

- This image comes from the Login.gov Design System, but is not used within the design system itself, so I'm aiming to remove it from there
- Avoids redundancy and reduces risk for desync in the styling of "Close" icons
- May marginally improve performance since the single `sprite.svg` SVG sprite image is assumed to be cached by the time this image is displayed

## 📜 Testing Plan

1. Go to http://localhost:3000 on a mobile device (or emulated browser viewport)
2. Sign in
3. From the account dashboard, click "Menu" in the top-right
4. Observe that the menu's close button is visible, shown as a blue "X" icon

## 👀 Screenshots

Visual differences are expected to be very minimal.

Before|After
---|---
![localhost_3000_account(iPhone XR) (1)](https://github.com/18F/identity-idp/assets/1779930/b0b52b94-4568-4139-a338-769ce05010ff)|![localhost_3000_account(iPhone XR)](https://github.com/18F/identity-idp/assets/1779930/dec1fd26-d58b-487c-976b-c3c6f394f22a)
